vmdb/spec/models/orchestration_template_hot_spec.rb

```diff
@@ -1,18 +1,28 @@
 require "spec_helper"
 
-describe OrchestrationTemplate do
-  let(:sample) do
-    'spec/fixtures/orchestration_templates/hot_parameters.yml'
+describe OrchestrationTemplateHot do
+  describe ".eligible_manager_types" do
+    it "lists the classes of eligible managers" do
+      OrchestrationTemplateCfn.eligible_manager_types.each do |clss|
+        clss.is_subclass_of?(EmsCloud).should be_true
```

Comment:
Hot templates are only eligible to be provisioned on Openstack providers so this test needs to be more specific then just EmsCloud derived classes.